docs/articles/introduction-to-IoT-security.md

```diff
@@ -0,0 +1,117 @@
+
+![Generic IoT Architecture][generic-iot-model]
+
+Each of the elements in this model expose various vulnerabilities that you need to address.
+
+### Sources of device vulnerabilities
+
+Devices will be deployed to observe or control processes in environments that might not provide any significant protection against physical tampering or theft. An attacker could potentially replace the device with a spoof that captures sensitive information and leaks it to a third party or that operates maliciously. The replacement device could be programmed to stage a simple DoS attack by transmitting a large number of signals to the field gateway, effectively flooding that part of the system and rendering the field gateway from responding to requests from other devices or the control services.
+
+Even if a device is physically secured, spoofing and impersonation could still occur, especially if communications with a field gateway are wireless. Many devices are cheap commodity items that do not include the capability to encrypt data so it could be easy to introduce a rogue device that assumes the identity of a valid one into the local network and starts to present bad data. Additionally, a DoS attack could be staged by using a transmitter that interferes with the frequency used by the local wireless network without requiring any other form of entry into the system. Furthermore, it could be possible for an observer to silently intercept traffic, resulting in disclosure of sensitive or valuable information. Some form of shielding may be necessary.
+
+Devices that listen for incoming connections can be especially vulnerable unless these connections are authenticated in some way. Even then, a device can be made the target of a DoS attack simply by bombarding it with a large number of connection requests; even if these requests are recognized as invalid by the device, it can still be overwhelmed by the processing involved.
```

Comment:
This is very true. In fact, this is so universal that we _will_ give some specific guidance here. This is what @clemensv has named "Service Assisted Communication". If we introduce this risk here, let put in a placeholder that should link to the pattern. I've added an issue #168.